### DISCUSSION TITLE
getting error in deploying deploysimplestorage.s.sol file

### DISCUSSION THREAD

**Discussion by khushbusandhu** [Cyfrin/foundry-full-course-cu#553] [General]
i am getting this error while deployingsimplestorages.sol on sepolia test network
error is 
```
Error: 
error sending request for url (https://eth-sepolia.g.alchemy.com/v2/QDZeCFSfojeHnW_hrIEdMq8k0gB0d6o): error trying to connect: dns error: failed to lookup address information: Temporary failure in name resolution

Context:
- Error #0: error trying to connect: dns error: failed to lookup address information: Temporary failure in name resolution
- Error #1: dns error: failed to lookup address information: Temporary failure in name resolution
- Error #2: failed to lookup address information: Temporary failure in name resolution
- ```

**Comment by shikhar229169** [comment]
Can you restart your vs code and try again.

**Comment by khushbusandhu** [comment]
yes i did 
now i am getting this error
```-simple-storage-f23/foundry-simple-storage-f23/script/deploysimplestorage.s.sol --rpc-url $SEPOLIA_RPC_URL --private-key $PRIVATE_KEY --broadcast
error: a value is required for '--fork-url <URL>' but none was supplied

For more information, try '--help'.```

**Reply by shikhar229169** [reply]
You need to do source .env again
You need to do it everytime when you start a new terminal.

**Comment by khushbusandhu** [comment]
hey @shikhar229169 
still the same error is there
Error: 
```
error sending request for url (https://eth-sepolia.g.alchemy.com/v2/QDZeCFSfojeHnW_hrIEdMq8k0gB0d6o): error trying to connect: dns error: failed to lookup address information: Temporary failure in name resolution

Context:
- Error #0: error trying to connect: dns error: failed to lookup address information: Temporary failure in name resolution
- Error #1: dns error: failed to lookup address information: Temporary failure in name resolution
- Error #2: failed to lookup address information: Temporary failure in name resolution
- ```

**Reply by shikhar229169** [reply]
ping google.com

Is this working?

**Comment by khushbusandhu** [comment]
i am not getting any relavant content related to this from about an hour
if you know anything kindly help

**Comment by theirrationalone** [comment]
If you're on windows, it might happened when wsl asked for network permissions and you ignored...

**Reply by theirrationalone** [reply]
Or you at alchemy... 

Might created goerli Or mainnet's endpoint and trying to ping through sepolia endpoint... 

If not please delete sepolia from alchemy and create new one and then try again.

**Comment by khushbusandhu** [comment]
hey @theirrationalone can I continue on remix as  I am getting lot of trouble in foundry(wsl)?

**Reply by theirrationalone** [reply]
Remix is Great but i recommend you to face these challenges, Just using foundry is such an awesome skill.

You have full community support though..... 

Repetition is the Mother of any skill ~@PatrickAlphaC 

Just delete every thing... Wsl, its extension, project u created, and then try again.. 

Rewatch Patrick SIR... Reimplement stuffs, recreate development env. Setup... Try to triage things, ask chatGpt, ask us(community)... And at end Enjoy The Journey ✈️✈️✈️

**Comment by khushbusandhu** [comment]
okay
thanks for your words @theirrationalone it means alot to me at such a position where i am just getting started with this development world

**Comment by cromewar** [comment]
Try using a different alchemy project on you acc and be sure you have setup the RPC URL correctly.

**Comment by khushbusandhu** [comment]
I used the RPC_URL from infura.io and that worked successfully.
I don't  know whats the problem with sepolia RPC URL.

**Reply by theirrationalone** [reply]
I think it was the connection problem between your metamask and alchemy.
